tools/pb2py

```diff
@@ -38,14 +38,10 @@ def process_type(t, cls, msg_id, indexfile, is_upy):
             type = 'p.UVarintType'
             pytype = 'int'
 
-        elif v.type in (17,):
+        elif v.type in (17,18,):
```

Comment:
Please add a space after the first comma. Maybe also add tools/pb2py and other python tools that do not end with .py to `make style` target so flake8 will test them.